### PR TITLE
[GTK] Add support for ClickGestureRecognizer

### PR DESCRIPTION
**Pull request by vmartos**
### Description of Change ###

Add support for ClickGestureRecognizer

### Issues Resolved ### 
None

### API Changes ###
None

### Platforms Affected ### 

- GTK+

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### PR Checklist ###

- [ ] Has automated tests 
- [ ] Rebased on top of the target branch at time of PR
- [ ] Changes adhere to coding standard

### CODE REVIEW

**Comment by samhouts** [comment]
build

File: Xamarin.Forms.Platform.GTK/VisualElementTracker.cs [hunk 1]
```diff
@@ -349,21 +350,47 @@ private void OnContainerButtonPressEvent(object o, ButtonPressEventArgs args)
...
+					.GetGesturesFor<ClickGestureRecognizer>(g => g.NumberOfClicksRequired == numClicks &&
+															g.Buttons == ButtonsMask.Primary);
+
+				foreach (ClickGestureRecognizer recognizer in clickGestures)
+					recognizer.SendClicked(view, ButtonsMask.Primary);
+			}
+			else
+			{
+				// right click
+				IEnumerable<ClickGestureRecognizer> rigthClickGestures = view.GestureRecognizers
```
Comment (samhouts) [inline Xamarin.Forms.Platform.GTK/VisualElementTracker.cs]: rigthClickGestures >> rightClickGestures

**Comment by samhouts** [comment]
@jsuarezruiz Would you like to review this one? Thanks!

File: Xamarin.Forms.Platform.GTK/VisualElementTracker.cs (hunk 1 above)
Comment (vmartos) [reply Xamarin.Forms.Platform.GTK/VisualElementTracker.cs]: changed and rebased on top of master

**Comment by vmartos** [comment]
Hi @jsuarezruiz ,

I added a ClickGestureGalleryPage in order to test ClickGestures. I do not have problems with transparencies in my branch.

click_gestures

**Comment by vmartos** [comment]
Hello @samhouts, any update on this pull request? @jsuarezruiz already approved. I can rebase to master if needed or squash both commits into one.

**Comment by samhouts** [comment]
build

**Pull request merged**